app/controllers/v1/users_controller.rb

```diff
@@ -151,4 +158,8 @@ def regenerate_otp_provisioning_uri!
   def remove_password_from_params_when_blank?
     params[:data][:attributes].delete(:password) if params[:data][:attributes][:password].blank?
   end
+
+  def nextcloud_groups
+    current_user.active_groups.map(&:id).join(',')
```

Comment:
Willen we voor alle groepen Nextcloud ondersteuning hebben, of alleen voor bijvoorbeeld besturen en commissies

Reply:
Het lijkt me voor de complexiteit zelf wel makkelijk om het voor alle groepen te doen

Reply:
Nu synchroniseert hij ze allemaal, wat we er dan verder mee doen in nextcloud kunnen we dan nog bepalen. Dus alle groepen is het makkelijkste.